src/common/Icons.js

```diff
@@ -51,6 +51,7 @@ const makeIcon = <Glyphs: string>(
   return <FixedIcon name={name} {...props} />;
 };
 
+export const IconBrowser = makeIcon(Feather, 'chrome');
```

Comment:
Another thing that I'd like to improve is the icon - I'm using the
    Feather icon for Google Chrome, since the only other browser-related
    icons I could find were the globe icon we're using for the language
    setting in this same screen, and the compass icon, which is less
    recognizable than the Chrome one. I'm unhappy with using a branded icon
    here, but Feather doesn't give us very many options, sadly.

Hmm. Yeah, this icon isn't very appropriate:
* It looks like it's suggesting the setting means something like "use Chrome". But actually its effect is independent of whether you use Chrome or some other browser, or at least it is on Android -- with either value of this setting, you get the same choice of browsers when you open a link.
* For most iOS users, with Safari as their browser, it doesn't much match what the setting does at all.

It might be best to just leave out the icon. In general, settings UIs often have icons for high-level groups of settings, but usually don't for individual settings.